blueprints/ember-redux/files/__root__/reducers/index.js

```diff
@@ -0,0 +1,18 @@
+import { combineReducers } from 'redux';
+
+/**
+ * This is the master reducer, it partitions the jobs amounts
+ * one or more other reducers which take on responsibility for
+ * a discrete part of the state tree.
+ *
+ * Use the "ember generate reducer" command to create additional
+ * reducers.
+ */
+
+// import foo from './foo';
+// import bar from './bar';
+
+export default combineReducers({
```

Comment:
> **Note: ** wrt to the `ember.d.ts` you're seeing this because anyone who uses vs-code will also automatically get typings for Ember and so fits into the same/similar category to the other vs-code config files ... for this one I think we should definitely include `typings/*` in the .gitignore. My bad for letting this through.